cypress.json

```diff
@@ -1,3 +1,4 @@
 {
-  "baseUrl": "http://localhost:3000"
+  "baseUrl": "http://localhost:3000",
+  "chromeWebSecurity": false
```

Comment:
I turned this off for now to prevent CORS errors. I'm still not sure why we're receiving CORS errors though. I thought it might have been the giphy image, but locally it runs fine. This should be revisited. See [Web Security](https://docs.cypress.io/guides/guides/web-security.html#Limitations) in the official Cypress docs.